ui-kit/src/main/res/values/white_palette.xml

```diff
@@ -12,12 +12,14 @@
     <color name="color_green">#2DB533</color>
     <color name="color_green_transparent">#802DB533</color>
 
+    <color name="night">#2B2B2B</color>
+    <color name="night_60">#992B2B2B</color>
```

Comment:
new colors in uikit??

Reply:
we do not have alpha but we use it in UI :(